src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -140,7 +140,9 @@ private async Task<(string name, DataType type, object value)[]> BindInputsAsync
                     return (binding.Metadata.Name, bindingContext.DataType, bindingContext.Value);
                 });
 
-            return await Task.WhenAll(bindingTasks);
+            var result = await Task.WhenAll(bindingTasks);
```

Comment:
nit: can you please change from `var` to `Task` here?

Reply:
I think it's not `Task` - it's `(string name, DataType type, object value)[]`.